CHANGELOG.MD

```diff
@@ -1,4 +1,95 @@
+- The minimum required runtime of the full .Net Framework was raised from `4.5.1` to `4.5.2` as `4.5.1` is not supported any more. If the OS is patched then Windows update will have ensured that the newer runtime patch is available. (#1108) (by @bergmeister)
+
+### Formatter Rules
+
+- Add `UseCorrectCasing` formatting rule for cmdlet/function name (#1117) (by @bergmeister)
+- Fix `PSUseConsistentIndentationRule` to handle pipes correctly when there is a multi-line statement after a pipe and add `PipelineIndentation` customisation option for it (#1102) (by @bergmeister)
+- Add `CheckInnerBrace` and `CheckPipe` options to `PSUseConsistentWhitespace` (#1092) (by @bergmeister)
+
+### Script Analysis Rules
+
+- Add command data file for PowerShell `2.0` (#1091) (by @bergmeister)
+- Add Command data files of PowerShell `6.1.0` (#1075) (by @bergmeister)
+- New PowerShell compatibility rules: `UseCompatibleCommands`, `UseCompatibleSyntax` and `UseCompatibleTypes` (#1156 and #1133) (by @rjmholt)
+- Make `PossibleIncorrectComparisonWithNull` rule return a `SuggestCorrection` for auto-fixes in VS-Code or via the -Fix switch (#1115) (by @bergmeister)
+- Make UseCompatibleCmdlets not throw if default reference desktop-5.1.14393.206-windows is specified in the list of platforms and use core-6.1.0-windows as an alternative default reference (#1043 and #1075) (by @bergmeister)
+
+### Custom Rules
+
+- Translate `RuleSuppressionID` for custom rules as well (#1144) (by @bergmeister)
+- Allow for suppression of custom rules (#1145) (by @bergmeister)
+
+### DSC
+
+- Add support for Hashtable syntax of `Import-DscRessource` for `-SaveDscDependency` switch (#1138) (by @bergmeister)
+- Take module version into account for `-SaveDscDependency` switch (#1094) (by @bergmeister)
+
+### Performance
+
+Multi-threading efficiency was highly improved leading to a speedup whilst keeping the CPU usage roughly the same. One can expect `PSScriptAnalyzer` (PSSA) `1.18` to be more than twice as fast in most situations (including 'cold' runs) compared to its predecessor 1.17.0.
+
+- Increase lock granularity for CommandInfo cache #1166 (by @bergmeister and @rjmholt)
+- Restore caching in Helper.GetCommandInfo (#1074) (Thanks @SeeminglyScience!)
+
+### Fixes and General Improvements
+
+- Fix bug when culture was set to Turkish (#1095) (by @bergmeister)
+- Fix settings file array parsing when no commas are present (#1161) (by @rjmholt)
+- Ensure correct disposal with using statement (#1164) (by @bergmeister)
+- Emit parsing errors as diagnostic records (#1130) (by @JamesWTruher)
+- Make Invoke-Formatter cleanup its state in the same way as Invoke-ScriptAnalyzer does to avoid knock on effect on Invoke-ScriptAnalyzer when Invoke-Formatter is being called (#1121) (by @bergmeister)
```

Comment:
Maybe clarify this item -- I'm not quite sure what it means

Reply:
Yes, you're right. The name of that PR was not good. Basically Invoke-ScriptAnalyzer did not do anything the first time after having called Invoke-Fornatter